docs/conf.py

```diff
@@ -398,3 +398,19 @@ def setup(app):
     f.write(zenodo_record)
 
 print(zenodo_record)
+
+
+# -- Creating tutorials.rst ---------------------------------------------------
+
+notebooks = ""
+
+for root, dirs, fnames in os.walk("io/"):
+    for fname in fnames:
+        if fname.endswith(".ipynb") and "checkpoint" not in fname:
+            notebooks += "\n* :doc:`" + root + "/" + fname[:-6] + "`"
```

Comment:
```suggestion
            notebooks += f"\n* :doc:`{root}/{fname[:-6]}`"
```
Doesn't f-strings look neater also easier to write than string concatenation :smile: